test/IntelFPGAFunctionAttributes.ll

```diff
@@ -94,7 +97,8 @@ attributes #3 = { nounwind }
 !5 = !{i32 8}
 !6 = !{i32 1, i32 1, i32 1}
 !7 = !{i32 1}
-!8 = !{!9, !9, i64 0}
-!9 = !{!"any pointer", !10, i64 0}
-!10 = !{!"omnipotent char", !11, i64 0}
-!11 = !{!"Simple C++ TBAA"}
+!8 = !{i32 0}
```

Comment:
Is it possible to get LLVM IR with `uses_global_work_offset` metadata equal to `1`?

Reply:
No, this SPIR-V attribute is mapped only to `uses_global_work_offset` equal to `0`

Reply:
I mean what if you get input LLVM IR (which needs to be translated into SPIR-V), where `uses_global_work_offset` metadata is equal to `1` or something else?

From the updated test I see that FE compiler you are using doesn't generate this metadata at all if non-zero argument was specified, but this might not be the case in the future or with other FE compiler.

Reply:
I see, will add the check for not `0` value

Reply:
As far as metadata doesn't have arguments with the last compiler changes, I mark this conversation as resolved